docs/en/status-communities/change-a-token-gated-community-to-open.md

```diff
@@ -0,0 +1,43 @@
+---
+id: 637
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Change a token-gated community to open
+
+!!! note ""
+    Currently, you can only set up token-gated and open communities using Status desktop. [:octicons-desktop-download-16: Download Status desktop][status-web-download].
+
+In a [token-gated community][token-based-access-to-communities-and-channels] members should meet specific token requirements to join. As a community owner, you can change an open-gated community into an open one so anyone can join without requiring tokens.
+
+!!! note
+    In addition to token requirements, you can set up [manual approval][about-community-request-approvals] to join your community. Token requirements and manual approval work independently.
+
+You might want to transition from a token-gated community to an open one for various reasons. For instance, you might want to grow your community by attracting new members, or perhaps you want to make your community more inclusive by removing the entry barrier.
+
+!!! tip
+    You can use token-gated communities and channels separately. For example, you may want to set up an open Status Community but [restrict access to certain channels][set-up-a-token-gated-channel].
+
+## What to expect
+
+- Remove all the `Become member` community permissions to transform your token-gated community into an open one.
+- You can change your community from token-gated to open or the other way around at any time.
```

Comment:
If I own an open community and change it to token-gated, will current members be affected by this?

Reply:
Yes, members not meeting the community requirements will be kicked out. I have simplified the sentence to reference only the change from token-gated to open because the article should focus on this. The corresponding articles already explain all the nuances of changing from open to token-gated.